common/app/experiments/Experiments.scala

```diff
@@ -71,3 +72,14 @@ object ServerSideLiveblogInlineAds
       sellByDate = LocalDate.of(2023, 6, 1),
       participationGroup = Perc0C,
     )
+
+object PoorDeviceConnectivity
+    extends Experiment(
+      name = "poor-device-connectivity",
+      description = "Serve a pared back version of the site to users experiencing poor device connectivity",
```

Comment:
just because 'pared back version' could imply an alternate site:

```suggestion
      description = "Reduce the load of the site for users experiencing poor device connectivity",
```